lib/chef/knife/windows_cert_generate.rb

```diff
@@ -56,6 +56,12 @@ class WindowsCertGenerate < Knife
         :long => "--cert-passphrase PASSWORD",
         :description => "Password for certificate."
 
+      option :overwrite_certs,
```

Comment:
I actually don't think we need this flag -- it's something we could add back later if users ask for it.